server/api/impl/admin/install.go

```diff
@@ -6,27 +6,33 @@ package admin
 import (
 	"fmt"
 	"net/http"
+	"strings"
 
 	"github.com/pkg/errors"
 
 	"github.com/mattermost/mattermost-server/v5/model"
 
 	"github.com/mattermost/mattermost-plugin-apps/apps"
 	"github.com/mattermost/mattermost-plugin-apps/server/api"
-	"github.com/mattermost/mattermost-plugin-apps/server/utils"
 	"github.com/mattermost/mattermost-plugin-apps/server/utils/md"
 )
 
 func (adm *Admin) InstallApp(cc *apps.Context, sessionToken apps.SessionToken, in *apps.InInstallApp) (*apps.App, md.MD, error) {
 	// TODO <><> check if acting user is a sysadmin
 
-	app, err := adm.store.App().Get(cc.AppID)
+	m, err := adm.store.Manifest().Get(cc.AppID)
 	if err != nil {
 		return nil, "", err
 	}
 
-	app.GrantedPermissions = in.GrantedPermissions
-	app.GrantedLocations = in.GrantedLocations
+	app, _ := adm.store.App().Get(cc.AppID)
```

Comment:
I'd argue that app should come with the manifest attached already. Or we could add `getWithManifest`/`getWithoutManifest` methods for more clarity

Reply:
Not sure what this means. An `App` embeds a `Manifest`.